packages/react/src/Button/Button.tsx

```diff
@@ -6,26 +6,31 @@
 import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { ButtonHTMLAttributes, ForwardedRef, PropsWithChildren } from 'react'
+import { Icon } from '../Icon'
 
 export type ButtonProps = {
   /** The level of prominence. Use a primary button only once per page or section. */
   variant?: 'primary' | 'secondary' | 'tertiary'
+  icon?: Function
+  iconPosition?: 'start' | 'only'
```

Comment:
Should we do ‘before’ instead of ‘start’?

Reply:
I don't think so I've been using the inline/block start/end terminology everywhere. Before reminds me of pseudo elements instead of logical properties.